packages/w/workrave/pspec_x86_64.xml

```diff
@@ -165,26 +158,89 @@
+            <Path fileType="data">/usr/share/gnome-shell/extensions/workrave@workrave.org/metadata.json</Path>
+            <Path fileType="data">/usr/share/gnome-shell/extensions/workrave@workrave.org/stylesheet.css</Path>
+        </Files>
+    </Package>
+    <Package>
+        <Name>mate-workrave-applet</Name>
+        <Summary xml:lang="en">MATE panel applet for Workrave</Summary>
+        <Description xml:lang="en">Workrave is a free program that assists in the recovery and prevention of Repetitive Strain Injury (RSI). It monitors your keyboard and mouse usage and using this information, it frequently alerts you to take microbreaks, rest breaks and restricts you to your daily computer usage.
+</Description>
+        <PartOf>desktop.mate</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="13">workrave</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="library">/usr/lib64/mate-applets/workrave-applet</Path>
+            <Path fileType="data">/usr/share/dbus-1/services/org.mate.panel.applet.WorkraveAppletFactory.service</Path>
+            <Path fileType="data">/usr/share/mate-panel/applets/org.workrave.WorkraveApplet.mate-panel-applet</Path>
+            <Path fileType="data">/usr/share/mate-panel/ui/workrave-menu.xml</Path>
+        </Files>
+    </Package>
+    <Package>
+        <Name>xfce4-workrave-applet</Name>
+        <Summary xml:lang="en">XFCE4 panel applet for Workrave</Summary>
+        <Description xml:lang="en">Workrave is a free program that assists in the recovery and prevention of Repetitive Strain Injury (RSI). It monitors your keyboard and mouse usage and using this information, it frequently alerts you to take microbreaks, rest breaks and restricts you to your daily computer usage.
+</Description>
+        <PartOf>desktop.xfce</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="13">workrave</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="library">/usr/lib64/xfce4/panel/plugins/libworkrave-plugin.so</Path>
+            <Path fileType="data">/usr/share/xfce4/panel/plugins/workrave-xfce-applet.desktop</Path>
+        </Files>
+    </Package>
     <Package>
         <Name>workrave-devel</Name>
         <Summary xml:lang="en">Development files for workrave</Summary>
         <Description xml:lang="en">Workrave is a free program that assists in the recovery and prevention of Repetitive Strain Injury (RSI). It monitors your keyboard and mouse usage and using this information, it frequently alerts you to take microbreaks, rest breaks and restricts you to your daily computer usage.
 </Description>
         <PartOf>programming.devel</PartOf>
         <RuntimeDependencies>
-            <Dependency release="12">workrave</Dependency>
+            <Dependency release="13">gnome-shell-extension-workrave</Dependency>
```

Comment:
I don't understand why this is being added.

Reply:
```
- ^gnome-shell-extension-workrave :
        - /usr/lib64/girepository-1.0/Workrave-2.0.typelib
        - /usr/lib64/libworkrave-gtk4-private-1.0.so.* # Maybe because this is here ?
        - /usr/share/gnome-shell/extensions/workrave@workrave.org/*
```